workflows/data_pipelines/ess_france/config.py

```diff
@@ -10,8 +10,12 @@
     tmp_folder=f"{DataSourceConfig.base_tmp_folder}/ess",
     minio_path="ess",
     file_name="ess",
-    resource_id="57bc99ca-0432-4b46-8fcc-e76a35c9efaf",
-    url=f"{DATA_GOUV_BASE_URL}57bc99ca-0432-4b46-8fcc-e76a35c9efaf",
+    files_to_download={
+        "ess": {
```

Comment:
Same here. It seems like a needless repetition.

Reply:
See in https://github.com/annuaire-entreprises-data-gouv-fr/search-infra/pull/441#discussion_r1895532423